miasm2/jitter/jitcore_cc_base.py

```diff
@@ -88,14 +88,14 @@ def init_codegen(self, codegen):
     def label2fname(self, label):
```

Comment:
`label2fname` is now more something like `loc_key_to_filename`

Reply:
done